components/src/core/components/SchemaForm/SchemaForm.vue

```diff
@@ -157,6 +160,7 @@ export default defineComponent({
                 key: field.key,
                 label: $t(field.label),
                 labelHelpText : $t(field.labelHelpText),
+                disabled: isDisabled.value,
```

Comment:
```suggestion
                disabled: props.schema?.disabled === true,
```

Reply:
can add the same to `createActionNode` and get rid of computed property

Reply:
@Super-Chama corrected it. Thanks